app/models/opendap_configuration.rb

```diff
@@ -160,22 +160,30 @@ def info_urls_for(granule)
     end
   end
 
-  def download_urls_for(granule)
+  def download_urls_for(granule, is_cwic=false)
```

Comment:
There should be a separate `CwicUrlMapper` class instead of using if-logic to overwrite methods.